docker-compose.yml

```diff
@@ -300,6 +300,7 @@ services:
   api:
     image: pecan/api:${PECAN_VERSION:-latest}
     user: "${UID:-1001}:${GID:-1001}"
+    restart: unless-stopped
```

Comment:
It can be useful to automatically restart the api container if it fails when we bring up the whole docker stack